quarkus-workshop-super-heroes/super-heroes/rest-fights/src/test/java/io/quarkus/workshop/superheroes/fight/FightResourceTest.java

```diff
@@ -67,8 +85,10 @@ public void testHelloEndpoint() {
     void shouldNotGetUnknownFight() {
         Long randomId = new Random().nextLong();
         given()
-            .pathParam("id", randomId)
-            .when().get("/api/fights/{id}")
+            .pathParam("id",
```

Comment:
Here it does have some advantages, because everything after it is indented a few spaces further right if the line break after given() isn't there. But I've removed.